packages/now-routing-utils/test/superstatic.spec.js

```diff
@@ -181,6 +181,15 @@ test('convertRedirects', () => {
       source: '/hello/:world*',
       destination: '/something#:world*',
     },
+    {
+      source: '/external/:id',
+      destination:
+        'https://example.com/?utm_source=google.com#/guides/:id/page?dynamic=code',
```

Comment:
A URL like this really works?

Reply:
Yes, in fact you can visit that one in your browser without any issues. And then do `console.log(document.location.hash)` and see the hash is preserved.